container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -27,18 +28,34 @@ const (
 	http_listen {{.Service.HttpServerHost}}
 	http_port {{.Service.HttpServerPort}}
 	storage.path {{.Service.StoragePath}}
+	parsers_file /fluent-bit/etc/parsers.conf
```

Comment:
There are two `parsers_file` entries. Is this a bug, where the second one is overriding the first? Or should there be two?

Reply:
fluentbit allows defining multiple parsers files it will just concatenate the parsers from all of them , no overriding will happen (unless two have conflicting names likely)